src/psyclone/transformations.py

```diff
@@ -3339,23 +3604,26 @@ def name(self):
         '''
         return "ACCKernelsTrans"
 
-    def apply(self, node_list, default_present=False):
+    def apply(self, node_list, default_present=False, options=None):
```

Comment:
Should `default_present` now be supplied via the `options` dict?

Reply:
Dang, of course it should. Should be all fixed now.